shared/product-make.include

```diff
@@ -88,7 +88,7 @@ $(OUT)/bash-remediations.xml: $(SHARED)/$(TRANS)/combineremediations.py $(bash_r
 	$(SHARED)/$(TRANS)/combineremediations.py $(PROD) $(BUILD_REMEDIATIONS) $(OUT)/bash-remediations.xml
 
 # Common build targets - an XCCDF with remediations but not linked to OVAL yet
-$(OUT)/xccdf-unlinked-withremediations.xml: $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/xccdf-unlinked-ocilrefs.xml $(OUT)/bash-remediations.xml $(SHARED)/$(TRANS)/xccdf-addremediations.xslt
+$(OUT)/xccdf-unlinked-withremediations.xml: $(OUT)/xccdf-unlinked-ocilrefs.xml $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/bash-remediations.xml $(SHARED)/$(TRANS)/xccdf-addremediations.xslt
```

Comment:
Hi Martin,

@mpreisler 
> Even if we keep it at this stage this whole hunk does nothing.

That's not completely true. Let me explain below.

> It used to be X has to be newer than A and B. You changed it to X has to be newer than B and A.

Correct. But this is not the sole effect of that change (and actually isn't the motivation why that change was performed).

What the change does also is it changes the source file, from which the ```$(OUT)/xccdf-unlinked-withremediations.xml``` intermediary target will be created. Before first prerequisite (```$<```) was set to ```$(OUT)/xccdf-unlinked-resolved.xml```. The aforementioned change is moving it to be ```$(OUT)/xccdf-unlinked-ocilrefs.xml``` just by flipping the order of prerequisites (I didn't want to delete the other ```$(OUT)/xccdf-unlinked-resolved.xml``` just for the case it might come handy in the future due some reason).

This automatic Makefile variables syntax:
&nbsp; &nbsp; [1] https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

was already present as convention in ```shared/product-make.include``` thus when making this change I didn't want to move away from it and use direct ```$(OUT)/ocil-unlinked.xml``` reference / name instead.

But I will add a comment (probably even replace ```$<``` automatic variable with explicit name of first prerequisite) so the purpose of this change is more explicit / clear.

> That's my question.

Thanks for explanation. Makes sense now.

Jan.